perllib/FixMyStreet/DB/Result/Body.pm

```diff
@@ -290,13 +290,16 @@ granted a specific permission.
 
 sub staff_with_permission {
     my ( $self, $permission ) = @_;
-    return $self->users->search([
+    my @staff = $self->users->search([
         { 'user_body_permissions.permission_type' => $permission },
         { permissions => \"@> ARRAY['$permission']" }
     ], {
         join => [ 'user_body_permissions', { "user_roles" => "role" } ],
         order_by => { '-asc' => ['name'] },
```

Comment:
Is there a reason not to use `distinct => 1` here and do the de-duplication in the database?

Reply:
Nope - far better if I've got that right.